### PR TITLE
cloud: add doc for sql editor

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-tidb-crud-sql.md [hunk 1]
```diff
@@ -19,7 +19,23 @@ Please make sure you are connected to a TiDB cluster. If not, refer to [Build a
 
 TiDB is compatible with MySQL, you can use MySQL statements directly in most cases. For unsupported features, see [Compatibility with MySQL](/mysql-compatibility.md#unsupported-features).
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try the SQL editor in your TiDB Cloud console. Navigate to the **Cluster Details** page and click **SQL Editor** on the left navigation bar. In the SQL editor, you can edit and run SQL queries directly against your cluster without using a terminal.
+
+> **Note:**
+>
+> The SQL editor currently has limited support on SQL. DDLs such as `CREATE TABLE` or `DROP TABLE` are not supported yet.
```
Comment (Oreoxmt) [inline develop/dev-guide-tidb-crud-sql.md:26]:
```suggestion
> The SQL editor currently has limited support on SQL statements. DDLs such as `CREATE TABLE` or `DROP TABLE` are not supported yet.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 2]
```diff
@@ -69,82 +69,52 @@ After trying **Plaground**, you can load sample data to your TiDB Cloud cluster.
...
-    {{< copyable "sql" >}}
+    * Press **Ctrl + Shift + Enter**.
+    * Select the two queries with your cursor and click **Run**.
 
-    ```sql
-    SELECT * FROM trips WHERE start_station_name = '8th & D St NW';
-    ```
-
-5. Show the least ten popular bicycle stations for picking up:
+    In the query log box, you can see the two queries are executed one by one.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:110]:
```suggestion
    In the query log panel, you can see the two queries are executed one by one.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 3]
```diff
@@ -69,82 +69,52 @@ After trying **Plaground**, you can load sample data to your TiDB Cloud cluster.
...
-    {{< copyable "sql" >}}
+5. To view the tables in the `bikeshare` database, type in the following SQL statement in the editor:
 
     ```sql
-    USE bikeshare;
     SHOW tables;
     ```
 
-2. Check the structure of the `trip` table:
+    To run the query, press **Ctrl + Enter** or click **Run**. You can see the query log and results immediately on the bottom of the page.
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply tidb-cloud/tidb-cloud-quickstart.md:96]: How about using ▶️?

[the remark this answers is not in the document]
Comment (ran-huang) [reply tidb-cloud/tidb-cloud-quickstart.md:96]: forget about it. It's too ugly

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 4]
```diff
@@ -69,82 +69,52 @@ After trying **Plaground**, you can load sample data to your TiDB Cloud cluster.
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply tidb-cloud/tidb-cloud-quickstart.md:69]: L34: **After the creation is completed**, perform security settings for your cluster:

Other suggestions are accepted. Thanks!

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 5]
```diff
@@ -69,82 +69,48 @@ After trying **Plaground**, you can load sample data to your TiDB Cloud cluster.
...
 
-After loading data to the cluster, you can connect to your cluster from the command line or with a programming language.
+After loading data to the cluster, you can try running SQL statements directly from the console.
 
-1. Navigate to the **Clusters** page.
-
-2. In the area of your newly created cluster, click **Connect** in the upper-right corner. A connection dialog is displayed.
-
-3. Follow the instructions in the dialog to connect to your TiDB cluster.
+1. Hover on the left navigation bar and click **SQL Editor**. The SQL Editor page is displayed.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:76]:
```suggestion
1. Click **SQL Editor** on the left navigation bar. The SQL Editor page is displayed.
```

File: develop/dev-guide-tidb-crud-sql.md [hunk 6]
```diff
@@ -19,7 +19,23 @@ Please make sure you are connected to a TiDB cluster. If not, refer to [Build a
 
 TiDB is compatible with MySQL, you can use MySQL statements directly in most cases. For unsupported features, see [Compatibility with MySQL](/mysql-compatibility.md#unsupported-features).
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try the SQL editor in your [TiDB Cloud console](https://tidbcloud.com/). Navigate to the **Cluster Details** page and click **SQL Editor** on the left navigation bar. In the SQL editor, you can edit and run SQL queries directly against your cluster without using a terminal.
```
Comment (Oreoxmt) [inline develop/dev-guide-tidb-crud-sql.md:22]:
```suggestion
To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try the SQL editor in your [TiDB Cloud console](https://tidbcloud.com/). Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and click **SQL Editor** on the left navigation bar. In the SQL editor, you can edit and run SQL queries directly against your cluster without using a terminal.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: develop/dev-guide-tidb-crud-sql.md (hunk 6 above)
Comment (ran-huang) [reply develop/dev-guide-tidb-crud-sql.md:22]: SQL editor is only available inside the individual cluster page.

**Pull request merged**